src/App.js

```diff
@@ -1,26 +1,37 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
```

Comment:
It isn't necessary to use data from a file .json

Reply:
I wanted to solve the task with the initial state